packages/devextreme/testing/tests/DevExpress.ui.widgets/gallery.markup.tests.js

```diff
@@ -78,6 +78,57 @@ QUnit.module('aria accessibility', () => {
         assert.equal($element.attr('aria-label'), 'gallery', 'widget should have aria-label to have difference from text list');
     });
 
+    QUnit.test('aria label for items', function(assert) {
```

Comment:
[minor, test naming]
```suggestion
    QUnit.test('aria label for items should be set', function(assert) {
```

Reply:
fixed